packages/cloud_firestore/cloud_firestore_web/lib/src/utils/exception.dart

```diff
@@ -7,13 +7,13 @@ import 'package:firebase_core_web/firebase_core_web_interop.dart'
     as core_interop;
 
 /// Returns a [FirebaseException] from a thrown web error.
-FirebaseException getFirebaseException(Object object) {
-  if (object is! core_interop.FirebaseError) {
-    return FirebaseException(
-        plugin: 'cloud_firestore', message: object.toString());
+Exception getException(Object object) {
+  if (object is! core_interop.FirebaseError && object is Exception) {
```

Comment:
What is the `is Exception` doing?

Reply:
Wouldn't we get a cast error if an `Error` is thrown instead?

Reply:
Can get rid of `is Exception`, I just wanted it to be a bit more explicit that `Object` was an `Exception`. But I guess it will be a problem if it is an `Error`. Will remove.